src/main/kotlin/com/openlattice/indexing/IndexingService.kt

```diff
@@ -90,9 +89,9 @@ class IndexingService(
 
     @Suppress("UNUSED")
     private val indexingWorker = executor.submit {
-        Stream.generate { indexingQueue.take() }
-                .parallel().forEach { entitySetId ->
-                    try {
+        while ( true ) {
+            try {
+                val entitySetId = indexingQueue.take()
```

Comment:
Formatting doesn't align.

Reply:
Yeah so this is to avoid erasing the history of that large stretch of code because I didn't change anything there, especially since this code doesn't change very often. I'm down to fix it if we all agree that I should but I don't see this as a blocking issue

Reply:
Please re-indent it, we can use `git blame -w` if we need to.

Reply:
Alternatively, extract it to a private function so the next time we change the looping structure the indentation doesn't change. :)

Reply:
Oh, awesome! I definitely didn't know about `blame -w`